### ISSUE TITLE
Extraction method for photos on the phone Apps?

### ISSUE THREAD

**Issue by eejd** [photosynth/offlineViewer#2]
In order for this to be useful (going forward) it would be great to allow us to continue to use the phone app and/or extract existing photos. Is there any plan or knowledge about how this could be accomplished?

**Comment by jayking1234** [comment]
Yes, where can one get an older version even to use on the iPhone?

**Comment by astrojonathan** [comment]
Can you be more specific about your use case with the Phone?

**Comment by jpickerell** [comment]
@astrojonathan, we would like to extract/view the photosynths that we took (or can still take if we have the app still installed) from our phones. Any ability for us to do that?

**Comment by MaxPalmer-UH** [comment]
I'm interested in the part of photosynth that enabled the relative orientation and location of photos to be determined. Is that open source or available anywhere?

**Comment by CurtisDS** [comment]
As of iOS 14.5 the Photosynth app will no longer run. This is unfortunate as I used it regularly. Unless Microsoft decides to update it to run on the newer OS... the app is dead. For those wondering if there is a way to extract panoramas from the app... there is. I used a program called iMazing which lets you navigate the file system of your iTunes phone backup. From there you can dump the panorama folder that is inside the app.

However these files are not in a format that are able to be used with this offline viewer. But at least you will have a backup of you panoramas and you can still look at the flattened.jpg files.

There is another piece of software called FSP Viewer that will allow you to import the flattened.jpg panoramas and view them... however it only really works with full panoramas. Any panoramas you took that were only partially captured will be warped by the viewer.
